python/tests/test_dbscan.py

```diff
@@ -166,13 +164,17 @@ def test_dbscan_numeric_type(gpu_number: int, data_type: str) -> None:
     ids=idfn,
 )
 @pytest.mark.parametrize("data_type", cuml_supported_data_types)
-@pytest.mark.parametrize("max_record_batch", [100, 10000])
+@pytest.mark.parametrize(
+    "max_record_batch", [pytest.param(100, marks=pytest.mark.slow), 10000]
+)
+@pytest.mark.parametrize("algorithm", ["brute", "rbc"])
```

Comment:
Is this test run time still reasonable?

Reply:
Yes, it's actually pretty fast given it runs on 4 GPUs.

Reply:
And that some parameters are marked as slow so not all of them are executed for CI.